CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ ENHANCEMENTS
 
 * `azurerm_kusto_cluster` - Support for `identity` [GH-7367]
 
+BUG FIXES: 
+
+* `azurerm_role_definition` - terraform import now sets scope to prevent a force recreate [GH-7424]
```

Comment:
Can we remove this change from the PR? Since the changelog changes often we commit these changes directly to master rather than including them in PR's to avoid conflicts